packages/flutter/lib/src/services/flutter_version.dart

```diff
@@ -15,7 +15,9 @@
 abstract final class FlutterVersion {
   const FlutterVersion._();
 
-  /// The Flutter version used to compile the app.
+  /// The value is a [semantic versioning](https://semver.org/) string representing the
+  /// version of the current Flutter version, possibly followed by whitespace and other
```

Comment:
"version of the current Flutter version" => "version of Flutter used to compile the app"

Reply:
```suggestion
  /// version of Flutter used to compile the app, possibly followed by whitespace and other
```